app/views/shared/_individual_progress.html.haml

```diff
@@ -40,7 +40,7 @@
 - if step == 1 && search_page == true
   = link_to "Continue", insured_consumer_role_index_path(person: @person_params), method: :post, class: 'btn btn-lg btn-primary  btn-block'
 - elsif step == 1
-  %span{class: 'btn btn-lg btn-primary btn-br btn-block', onclick: 'PersonValidations.manageRequiredValidations($(this));'} CONTINUE
+  %span{tabindex: '0', id: 'continue_button', class: 'btn btn-lg btn-primary btn-br btn-block', onkeydown: 'handleButtonKeyDown(event, "continue_button");', onclick: 'PersonValidations.manageRequiredValidations($(this));'} CONTINUE
```

Comment:
Can we refactor this to `html.erb` like we did in the previous PR?

Reply:
I had originally refactored this, it just introduced a lot of cucumber test problems so I reverted. I can take a look again and see what parts would need to be changed.

Reply:
https://github.com/ideacrew/enroll/pull/3351/commits/046c87965fff2eec0a4c12c759819a55b67d9a0e